source/settings.rst

```diff
@@ -808,6 +808,14 @@ The settings that are used by Git are stored in the configuration files of Git.
 
     Enter the command to be run. This can be any command that your system can run e.g. an executable program,
     a .bat script, a Python command, etc. Use the ``Browse`` button to find the command to run.
+    
+    There are some special prefixes which change the way the script is executed:
+
+    * ``plugin:`` With this prefix, the text after it is interpreted as a name of a *plugin* (refer :ref:`settings-plugins`). If a plugin with that name is found, it is run.
+    * ``navigateTo:`` With this prefix, the text after it is interpreted as the actual script to run. After it finishes, the first line of its output is interpreted
+      as a commit hash and, if found, the corresponding commit is selected in the revision grid.
```

Comment:
Maybe mention `"(since v3.0)"` somewhere in this sentence. There's no harm in mentioning this, as even once we release 3.0 people will still use older versions.

Reply:
OK, good idea. Should I amend the original commit, or should I add a new one? I don't know what is the standard practice for updating PRs in the GE repositories.